cache/connpool.go

```diff
@@ -106,7 +106,9 @@ func (m *MemcacheConnPool) manager() {
 }
 
 func (m *MemcacheConnPool) connect() {
-	if c, err := memcache.Connect(m.server, DefaultTimeout); err != nil {
+	c := memcache.New(m.server)
```

Comment:
This new library has an internal conn pool, but cannot limit the maximum open connections.
I think it's okay to layer ours on top of it.

Reply:
I'll move concurrency limiting into a separate abstraction layer later.
Let me absorb this PR first.